installer/pkg/config-generator/tls.go

```
@@ -0,0 +1,71 @@
+package configgenerator
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/coreos/tectonic-installer/installer/pkg/validate"
+)
+
+// If not provided, TLS pem contents are empty
+var (
+	rootCACert = ""
+	rootCAKey  = ""
+)
+
+// GenerateTLSConfig fetches and validates the TLS cert files
+// If no file paths were provided, the certs will be auto-generated
+func (c *ConfigGenerator) GenerateTLSConfig() error {
```

This function suppresses all errors and always returns nil. How do we know if something is failing? We need to check all errors and surface them.